subgraphs/stargate/src/sdk/protocols/bridge/chainIds.ts

```diff
@@ -40,3 +41,7 @@ setChainID(11, Network.OPTIMISM);
 setChainID(111, Network.OPTIMISM);
 setChainID(12, Network.FANTOM);
 setChainID(112, Network.FANTOM);
+setChainID(51, Network.METIS);
+setChainID(151, Network.METIS);
```

Comment:
Where does this come from? I must be missing something https://chainlist.org/chain/1088